src/metrics/include/metrics/metrics.hpp

```diff
@@ -82,6 +129,7 @@ class RegistryT final {
     // OperationsMap is a map of
     // actor name -> operation name -> actor id -> OperationImpl (time series).
     using OperationsMap = std::unordered_map<std::string, OperationsByType>;
+    using CollectorsMap = std::unordered_map<std::string, v2::Collector>;
```

Comment:
Let's add a comment here saying that this is a map from "Actor.Operation.Phase" to collector